src/plugins/visualizations/public/wizard/show_new_vis.tsx

```diff
@@ -30,8 +29,12 @@ import {
   getUsageCollector,
   getApplication,
   getEmbeddable,
+  getDocLinks,
 } from '../services';
 
+// @ts-ignore
```

Comment:
What was the reason for this `@ts-ignore` here, since the imported file seems to be all typed correctly?

Reply:
My mistake 🙂